src/test/ui/const-generics/min_const_generics/const_fn_in_generics.rs

```diff
@@ -0,0 +1,19 @@
+// run-pass
+
+#![feature(min_const_generics)]
+#![allow(dead_code)]
+
```

Comment:
```suggestion
```
I don't think that's needed here, r=me after this. You may also want to update the issue description, but i don't care too much about that